client/app/pod/popular.controller.js

```diff
@@ -1,41 +1,37 @@
 angular.module('throwcast.popular')
 
-.controller('PopularController', function ($scope, $http) {
+.controller('PopularController', function ($scope, $http, popService) {
   $scope.h1 = 'Top Podcast';
-  $scope.podcasts;
   $scope.message;
   $scope.podcastLink;
   $scope.stationPodcasts;
-  $scope.flag = false;
   $scope.stationPodcastButton = 'Show Station Podcasts';
   //TODO: delete all reference to stations after mvp
   $scope.stations;
 
-  $scope.getPodcast = function () {
-    $http.get('http://localhost:8888/api/podcasts/').then( function (res) {
-      $scope.podcasts = res.data.data;
-    });
-  };
-  $scope.getPodcast();
+ $scope.getPodcast = function () {
```

Comment:
I don't this needs to be a function. just called the service and set the podcast in the then function.